test/test_replaygain.py

```diff
@@ -32,10 +32,9 @@
 except (ImportError, ValueError):
     GST_AVAILABLE = False
 
-if any(has_program(cmd, ['-v']) for cmd in ['mp3gain', 'aacgain']):
-    GAIN_PROG_AVAILABLE = True
-else:
-    GAIN_PROG_AVAILABLE = False
+GAIN_PROG_AVAILABLE = bool(
+    any(has_program(cmd, ['-v']) for cmd in ['mp3gain', 'aacgain'])
+)
```

Comment:
Not wild about this one—I know it's equivalent, but for constant declarations, I think it's nicer to see the check made explicitly.

Reply:
Fair enough, I've reverted the change.